### PR TITLE
Trackering all pages update

### PR DESCRIPTION
**Pull request by Astarsys**
Huge update. Like the last time watch commits only for the 25th of September. I will delete my fork again after it. 
Updated all files in the tracking tab. Used Fenik's updates and applied them to the rest of the files too. But not everywhere ./ links. Some links ../ because on the local version ./ links didn't work.

### CODE REVIEW

**Comment by fenik17** [comment]
@Astarsys can you resolve the conflicts, please?

**Comment by Astarsys** [comment]
conflicts resolved

File: docs/backend-api/resources/tracking/delivery.md [hunk 1]
```diff
@@ -9,93 +9,136 @@ API base path: `/delivery`.
...
-    "task" : ${task}, //a task object, for more info see task/ section
-    "tracker" : ${tracker}, //corresponding tracker object, for more info see tracker/ section
-    "restrictions": ${restrictions}, //tariff restrictions object, for more info see user/get_tariff_restrictions 
-    "first_name": "John", //first name of employee assigned to the task, or null if missing
-    "middle_name": "Micheel", //middle name of employee assigned to the task, or null if missing
-    "last_name": "Johnson", //last name of employee assigned to the task, or null if missing
-    "vehicle_label": "Service car 002", //label of the vehicle assigned to the task, or null if missing
-    "estimated_time": 1122 //estimated time of arrival in seconds, or null if unavailable
+    "user_id": 3,
+    "task" : {"task_object":},
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/delivery.md:49]: `{"task_object":}` - it's invalid example. Can you add a real example here or keep link to description?

File: docs/backend-api/resources/tracking/geocoder.md [hunk 2]
```diff
@@ -7,113 +7,161 @@ description: Search address and location using geocoder
 
 API path: `/geocoder`.
 
-### Geocoder types:
+## Geocoder types:
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/geocoder.md:10]: No need a colon at the end of caption.

File: docs/backend-api/resources/tracking/geocoder.md [hunk 3]
```diff
@@ -7,113 +7,161 @@ description: Search address and location using geocoder
 
 API path: `/geocoder`.
 
-### Geocoder types:
+## Geocoder types:
 
 Geocoder types:
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/geocoder.md:12]: Remove it, please.

File: docs/backend-api/resources/tracking/retranslator.md [hunk 4]
```diff
@@ -9,126 +9,264 @@ API path: `/retranslator`.
...
+    "login": "login",
+    "password": "password",
+    "enabled": true
 }
 ```
 
+* `id` - int. Retranslator ID.
+* `name` - string. Zone label.
+* `protocol_id` - int. Protocol ID.
+* `address` - string. Network address, e.g. "127.0.0.1 or localhost".
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/retranslator.md:52]: >"127.0.0.1 or localhost"

`127.0.0.1` or `localhost`.

File: docs/backend-api/resources/tracking/retranslator.md [hunk 5]
```diff
@@ -9,126 +9,264 @@ API path: `/retranslator`.
...
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/retranslator/update' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b", "retranslator": {"name": "Some server", "protocol_id": "123456", "address": "127.0.0.1", "port": 15000, "login": "proto", "password": "qewtyr", "enabled": true}}'
+    ```
+
+=== "HTTP GET"
+
+    ```
+    {{ extra.api_example_url }}/retranslator/update?hash=a6aa75587e5c59c32d347da438505fc3&retranslator={"name": "Some server", "protocol_id": 123456, "address": "127.0.0.1", "port": 15000, "login": "proto", "password": "qewtyr", "enabled": true}
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/retranslator.md:217]: This example won't work

File: docs/backend-api/resources/tracking/retranslator.md [hunk 6]
```diff
@@ -9,126 +9,264 @@ API path: `/retranslator`.
...
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/retranslator/create' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b", "retranslator": {"name": "Some server", "protocol_id": "123456", "address": "127.0.0.1", "port": 15000, "login": "proto", "password": "qewtyr", "enabled": true}}'
+    ```
+
+=== "HTTP GET"
+
+    ```
+    {{ extra.api_example_url }}/retranslator/create?hash=a6aa75587e5c59c32d347da438505fc3&retranslator={"name": "Some server", "protocol_id": 123456, "address": "127.0.0.1", "port": 15000, "login": "proto", "password": "qewtyr", "enabled": true}
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/retranslator.md:85]: This example won't work

File: docs/backend-api/resources/tracking/geocoder.md [hunk 7]
```diff
@@ -7,113 +7,161 @@ description: Search address and location using geocoder
...
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/geocoder/search_location' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b", "location": "{"lat": 56.827001, "lng": 60.594296}}'
+    ```
+
+=== "HTTP GET"
+
+    ```
+    {{ extra.api_example_url }}/geocoder/search_location?hash=a6aa75587e5c59c32d347da438505fc3&location={"lat": 56.827001, "lng": 60.594296}
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/geocoder.md:126]: This example won't work

File: docs/backend-api/resources/tracking/geocoder.md [hunk 8]
```diff
@@ -7,113 +7,161 @@ description: Search address and location using geocoder
...
-    "house": <string>, // optional
-    "postcode": <string>, // optional
-    "bounds": <bounds_object> //optional, the bounding box which can fully contain the returned result
-}
-```
+=== "HTTP GET"
 
-**bounds_object** is:
+    ```
+    {{ extra.api_example_url }}/geocoder/search_address?hash=a6aa75587e5c59c32d347da438505fc3&q=750 Avenue E,San Francisco,CA 94130,USA&lang=en&geocoder=google
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/geocoder.md:50]: This example won't work

File: docs/backend-api/resources/tracking/retranslator.md [hunk 9]
```diff
@@ -9,126 +9,264 @@ API path: `/retranslator`.
...
+      "fake_device_id_pattern": "id_pattern",
+      "required_login": true,
+      "required_password": false
+    }]
 }
 ```
 
+* `id` - int. Protocol ID.
+* `name` - string. Protocol name.
+* `has_login` - boolean. `True` if this protocol use login.
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/retranslator.md:268]: `True` -> `true`

File: docs/backend-api/resources/tracking/route/index.md [hunk 10]
```diff
@@ -5,42 +5,66 @@ description: Tracking route
...
+    curl -X POST '{{ extra.api_example_url }}/route/get' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b", "start": {"lat": 56.827001, "lng": 60.594296}, "end": {"lat": 52.835601, "lng": 60.514721}}'
+    ```
+
+=== "HTTP GET"
 
-Where **location** described in [data types description section](../../../getting-started.md#data-types).
+    ```
+    {{ extra.api_example_url }}/route/get?hash=a6aa75587e5c59c32d347da438505fc3&start={"lat": 56.827001, "lng": 60.594296}&end={"lat": 52.835601, "lng": 60.514721}
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/route/index.md:39]: This example won't work

File: docs/backend-api/resources/tracking/route/route_google.md [hunk 11]
```diff
@@ -1,81 +1,116 @@
...
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/route/google/get' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b", "start": {"lat": 56.827001, "lng": 60.594296}, "end": {"lat": 52.835601, "lng": 60.514721}}'
+    ```
+
+=== "HTTP GET"
+
+    ```
+    {{ extra.api_example_url }}/route/google/get?hash=a6aa75587e5c59c32d347da438505fc3&start={"lat": 56.827001, "lng": 60.594296}&end={"lat": 52.835601, "lng": 60.514721}
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/route/route_google.md:38]: This example won't work

File: docs/backend-api/resources/tracking/status/index.md [hunk 12]
```diff
@@ -1,100 +1,221 @@
...
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/status/create' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b", "listing_id": "12345", "status": {"label": "Busy", "color": "E57373"}}'
+    ```
+
+=== "HTTP GET"
+
+    ```
+    {{ extra.api_example_url }}/status/create?hash=a6aa75587e5c59c32d347da438505fc3&listing_id=12345&status={"label": "Busy", "color": "E57373"}
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/status/index.md:72]: This example won't work

File: docs/backend-api/resources/tracking/status/index.md [hunk 13]
```diff
@@ -1,100 +1,221 @@
...
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/status/update' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b", "status": {"id": "5", "label": "Busy", "color": "E57373"}}'
+    ```
+
+=== "HTTP GET"
+
+    ```
+    {{ extra.api_example_url }}/status/update?hash=a6aa75587e5c59c32d347da438505fc3&status={"id": "5", "label": "Busy", "color": "E57373"}
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/status/index.md:208]: This example won't work

File: docs/backend-api/resources/tracking/status/listing/index.md [hunk 14]
```diff
@@ -1,39 +1,84 @@
...
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/status/listing/create' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b", "listing": {"label": "Taxi driver statuses", "employee_controlled": "false", "supervisor_controlled": "true"}'
+    ```
+
+=== "HTTP GET"
+
+    ```
+    {{ extra.api_example_url }}/status/listing/create?hash=a6aa75587e5c59c32d347da438505fc3&listing={"label": "Taxi driver statuses", "employee_controlled": "false", "supervisor_controlled": "true"}
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/status/listing/index.md:35]: This example won't work

File: docs/backend-api/resources/tracking/status/listing/index.md [hunk 15]
```diff
@@ -42,32 +87,82 @@ Delete status listing.
...
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/status/listing/update' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b", "listing": {"id": "12345", "label": "Taxi driver statuses", "employee_controlled": "false", "supervisor_controlled": "true", "entries": [ 5, 2, 1, 4, 6]}'
+    ```
+
+=== "HTTP GET"
+
+    ```
+    {{ extra.api_example_url }}/status/listing/update?hash=a6aa75587e5c59c32d347da438505fc3&listing={"id": "12345", "label": "Taxi driver statuses", "employee_controlled": "false", "supervisor_controlled": "true", "entries": [ 5, 2, 1, 4, 6]}
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/status/listing/index.md:164]: This example won't work

File: docs/backend-api/resources/tracking/track/index.md [hunk 16]
```diff
@@ -8,21 +8,43 @@ description: Track
...
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/track/download' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b", "tracker_id": "123456", "from": "2020-09-23 03:24:00", "to": "2020-09-23 06:24:00", "format": "kml", "split": "false"}'
+    ```
+
+=== "HTTP GET"
+
+    ```
+    {{ extra.api_example_url }}/track/download?hash=a6aa75587e5c59c32d347da438505fc3&tracker_id=123456&from=2020-09-23 03:24:00&to=2020-09-23 06:24:00&format=kml&split=false
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/track/index.md:41]: This example won't work

File: docs/backend-api/resources/tracking/track/index.md [hunk 17]
```diff
@@ -32,130 +54,252 @@ _KML/KMZ file_ or _JSON response_ if requested time period exceeds limit specifi
...
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/track/list' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b", "tracker_id": "123456", "from": "2020-09-23 03:24:00", "to": "2020-09-23 06:24:00"}'
+    ```
+
+=== "HTTP GET"
+
+    ```
+    {{ extra.api_example_url }}/track/list?hash=a6aa75587e5c59c32d347da438505fc3&tracker_id=123456&from=2020-09-23 03:24:00&to=2020-09-23 06:24:00
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/track/index.md:93]: This example won't work

File: docs/backend-api/resources/tracking/track/index.md [hunk 18]
```diff
@@ -32,130 +54,252 @@ _KML/KMZ file_ or _JSON response_ if requested time period exceeds limit specifi
...
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/track/read' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b", "tracker_id": "123456", "from": "2020-09-23 03:24:00", "to": "2020-09-23 06:24:00"}'
+    ```
+
+=== "HTTP GET"
+
+    ```
+    {{ extra.api_example_url }}/track/read?hash=a6aa75587e5c59c32d347da438505fc3&tracker_id=123456&from=2020-09-23 03:24:00&to=2020-09-23 06:24:00
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/track/index.md:260]: This example won't work

File: docs/backend-api/resources/tracking/track/index.md [hunk 19]
```diff
@@ -32,130 +54,252 @@ _KML/KMZ file_ or _JSON response_ if requested time period exceeds limit specifi
...
+* `lat` - float.  Latitude.
+* `lng` - float.  Longitude.
+* `alt` - int. Altitude in meters. 
+* `satellites` - int. Number of satellites used in fix for this point.
+* `get_time` - string date/time. GPS timestamp of the point, in user's timezone.
+* `address` - string. Point address. Will be "" if no address recorded.
+* `heading` - int. Bearing in degrees (0..360).
+* `speed` - int. Speed in km/h.
+* `precision` - optional int. Precision in meters.
+* `gsm_lbs` - optional boolean. `True` if location detected by GSM LBS.
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/track/index.md:297]: True -> true

File: docs/backend-api/resources/tracking/track/waybill/index.md [hunk 20]
```diff
@@ -1,27 +1,53 @@
...
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/track/waybill/download' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b", "tracker_id": "123456", "from": "2020-09-23 03:24:00", "to": "2020-09-23 06:24:00", "type": "form3", "fill_history": "false", "number": "1234567"}'
+    ```
+
+=== "HTTP GET"
+
+    ```
+    {{ extra.api_example_url }}/track/waybill/download?hash=a6aa75587e5c59c32d347da438505fc3&tracker_id=123456&from=2020-09-23 03:24:00&to=2020-09-23 06:24:00&type=form3&fill_history=false&number=1234567
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/track/waybill/index.md:44]: This example won't work

File: docs/backend-api/resources/tracking/tracker/counter/value.md [hunk 21]
```diff
@@ -3,93 +3,127 @@ title: Counter Value
...
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}tracker/counter/value/list' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b", "trackers": [123456, 234567], "type": "odometer"}'
+    ```
+
+=== "HTTP GET"
+
+    ```
+    {{ extra.api_example_url }}tracker/counter/value/list?hash=a6aa75587e5c59c32d347da438505fc3&trackers=[123456, 234567]&type=odometer
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/counter/value.md:78]: This example won't work

File: docs/backend-api/resources/tracking/tracker/index.md [hunk 22]
```diff
@@ -541,9 +545,9 @@ Gets last point of the tracker located by GPS. Points located by GSM LBS are exc
     * `heading` - int. Direction bearing in degrees (0-360).
     * `speed` - int. Speed in km/h.
     * `precision` - int. Optional. Precision in meters.
-    * `gsm_lbs` - boolean. Optional. True if location detected by GSM LBS, optional.
-    * `parking` - boolean. Optional. true if point does not belong to track.
-    
+    * `gsm_lbs` - boolean. Optional. `True` if location detected by GSM LBS, optional.
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/index.md:548]: `True` -> `true`

File: docs/backend-api/resources/tracking/tracker/sensor/calibration_data.md [hunk 23]
```diff
@@ -11,52 +15,70 @@ Gets calibration data for sensor.
...
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/tracker/sensor/calibration_data/update' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b", "tracker_id": "123456", "sensor_id": "12345", "data": [{“in”:0.0,”out”:0.0},{“in”:0.7,”out”:60.0}]}'
+    ```
+
+=== "HTTP GET"
+
+    ```
+    {{ extra.api_example_url }}/tracker/sensor/calibration_data/update?hash=a6aa75587e5c59c32d347da438505fc3&tracker_id=123456&sensor_id=12345&data=[{“in”:0.0,”out”:0.0},{“in”:0.7,”out”:60.0}]
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/sensor/calibration_data.md:80]: This example won't work

File: docs/backend-api/resources/tracking/tracker/sensor/index.md [hunk 24]
```diff
@@ -3,84 +3,150 @@ title: Sensor actions
...
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/tracker/sensor/create' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b", "tracker_id": "123456", "sensor": {"type": "metering", "id": 860250,"sensor_type": "temperature", "name": "OBD Coolant temperature", "input_name": "obd_coolant_t", "divider": 1.0, "accuracy": 0, "units": "", "units_type": "celsius"}'
+    ```
+
+=== "HTTP GET"
+
+    ```
+    {{ extra.api_example_url }}/tracker/sensor/create?hash=a6aa75587e5c59c32d347da438505fc3&tracker_id=123456&sensor={"type": "metering", "id": 860250,"sensor_type": "temperature", "name": "OBD Coolant temperature", "input_name": "obd_coolant_t", "divider": 1.0, "accuracy": 0, "units": "", "units_type": "celsius"}
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/sensor/index.md:100]: This example won't work

File: docs/backend-api/resources/tracking/tracker/sensor/index.md [hunk 25]
```diff
@@ -89,36 +155,90 @@ Delete sensor with **sensor_id** from the database.
...
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/tracker/sensor/update' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b", "tracker_id": "123456", "sensor": {"type": "metering", "id": 860250,"sensor_type": "temperature", "name": "OBD Coolant temperature", "input_name": "obd_coolant_t", "divider": 1.0, "accuracy": 0, "units": "", "units_type": "celsius"}'
+    ```
+
+=== "HTTP GET"
+
+    ```
+    {{ extra.api_example_url }}/tracker/sensor/update?hash=a6aa75587e5c59c32d347da438505fc3&tracker_id=123456&sensor={"type": "metering", "id": 860250,"sensor_type": "temperature", "name": "OBD Coolant temperature", "input_name": "obd_coolant_t", "divider": 1.0, "accuracy": 0, "units": "", "units_type": "celsius"}
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/sensor/index.md:240]: This example won't work

File: docs/backend-api/resources/tracking/tracker/settings/special/index.md [hunk 26]
```diff
@@ -4,604 +4,901 @@ description: About special settings
...
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/tracker/settings/special/update' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b", "tracker_id": "123456" "value": {"type": "time_shift", "offset": 3.0}}'
+    ```
+
+=== "HTTP GET"
+
+    ```
+    {{ extra.api_example_url }}/tracker/settings/special/update?hash=a6aa75587e5c59c32d347da438505fc3&tracker_id=123456&value={"type": "time_shift", "offset": 3.0}
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/settings/special/index.md:900]: This example won't work

File: docs/backend-api/resources/tracking/tracker/settings/tracking.md [hunk 27]
```diff
@@ -1,48 +1,96 @@
...
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/tracker/settings/tracking/update' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b", "tracker_id": "123456", "tracking_settings": {"tracking_angle": 30, "tracking_distance": 100, "tracking_interval": 60, "on_stop_tracking_interval": 180, "sleep_mode": "disabled", "stop_detection": "ignition"}}'
+    ```
+
+=== "HTTP GET"
+
+    ```
+    {{ extra.api_example_url }}/tracker/settings/tracking/update?hash=a6aa75587e5c59c32d347da438505fc3&tracker_id=123456&tracking_settings={"tracking_angle": 30, "tracking_distance": 100, "tracking_interval": 60, "on_stop_tracking_interval": 180, "sleep_mode": "disabled", "stop_detection": "ignition"}
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/settings/tracking.md:79]: This example won't work

File: docs/backend-api/resources/tracking/tracker/stats/stats_engine_hours.md [hunk 28]
```diff
@@ -1,19 +1,42 @@
...
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/tracker/stats/engine_hours/read' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b", "tracker_id": "123456", "from": "2020-09-24 03:24:00", "to": "2020-09-24 06:24:00"}'
+    ```
+
+=== "HTTP GET"
+
+    ```
+    {{ extra.api_example_url }}/tracker/stats/engine_hours/read?hash=a6aa75587e5c59c32d347da438505fc3&tracker_id=123456&from=2020-09-24 03:24:00&to=2020-09-24 06:24:00
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/stats/stats_engine_hours.md:35]: This example won't work

File: docs/backend-api/resources/tracking/tracker/stats/stats_mileage.md [hunk 29]
```diff
@@ -1,19 +1,42 @@
...
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/tracker/stats/mileage/read' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b", "tracker_id": "123456", "from": "2020-09-24 03:24:00", "to": "2020-09-24 06:24:00"}'
+    ```
+
+=== "HTTP GET"
+
+    ```
+    {{ extra.api_example_url }}/tracker/stats/mileage/read?hash=a6aa75587e5c59c32d347da438505fc3&tracker_id=123456&from=2020-09-24 03:24:00&to=2020-09-24 06:24:00
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/tracker/stats/stats_mileage.md:35]: This example won't work

File: docs/backend-api/resources/tracking/zone/index.md [hunk 30]
```diff
@@ -5,142 +5,279 @@ description: Working with zones
...
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/zone/batch_convert' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b", "batch": "Geofence for test	Karlsplatz, 2"}'
+    ```
+
+=== "HTTP GET"
+
+    ```
+    {{ extra.api_example_url }}/zone/batch_convert?hash=a6aa75587e5c59c32d347da438505fc3&batch=Geofence for test	Karlsplatz, 2
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/zone/index.md:125]: This example won't work

File: docs/backend-api/resources/tracking/zone/index.md [hunk 31]
```diff
@@ -5,142 +5,279 @@ description: Working with zones
...
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/zone/create' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b", "zone": {"label": "Zone name", "address": "zone address", "radius": 100, "center": {"lat": 56.827001, "lng": 60.594296}}}'
+    ```
+
+=== "HTTP GET"
+
+    ```
+    {{ extra.api_example_url }}/zone/create?hash=a6aa75587e5c59c32d347da438505fc3&zone={"label": "Zone name", "address": "zone address", "radius": 100, "center": {"lat": 56.827001, "lng": 60.594296}}
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/zone/index.md:216]: This example won't work

File: docs/backend-api/resources/tracking/zone/index.md [hunk 32]
```diff
@@ -151,61 +288,117 @@ OR
...
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/zone/update' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b", "zone": {"label": "Zone name", "address": "zone address", "radius": 100, "center": {"lat": 56.827001, "lng": 60.594296}}}'
+    ```
+
+=== "HTTP GET"
+
+    ```
+    {{ extra.api_example_url }}/zone/update?hash=a6aa75587e5c59c32d347da438505fc3&zone={"label": "Zone name", "address": "zone address", "radius": 100, "center": {"lat": 56.827001, "lng": 60.594296}}
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/zone/index.md:366]: This example won't work

File: docs/backend-api/resources/tracking/zone/zone_point.md [hunk 33]
```diff
@@ -9,44 +9,92 @@ API base path: `/zone/point`.
...
+    ```shell
+    curl -X POST '{{ extra.api_example_url }}/zone/point/update' \
+        -H 'Content-Type: application/json' \ 
+        -d '{"hash": "22eac1c27af4be7b9d04da2ce1af111b", "zone_id": "1234567", "points": [{"lat": 11.0, "lng": 22.0, "node": true},{"lat": 11.2, "lng": 22.2, "node": true},{"lat": 11.4, "lng": 22.4, "node": true}]}'
+    ```
+
+=== "HTTP GET"
+
+    ```
+    {{ extra.api_example_url }}/zone/point/update?hash=a6aa75587e5c59c32d347da438505fc3&zone_id=1234567&points=[{"lat": 11.0, "lng": 22.0, "node": true},{"lat": 11.2, "lng": 22.2, "node": true},{"lat": 11.4, "lng": 22.4, "node": true}]
```
Comment (fenik17) [inline docs/backend-api/resources/tracking/zone/zone_point.md:96]: This example won't work

**Review by fenik17 — CHANGES_REQUESTED** [review]
@Astarsys Thanks! This is a very big job! Overall good. There are some comments. For example, the examples for `HTTP GET` should contain valid requests that can be applied in practice - all special characters must be escaped.

File: docs/backend-api/resources/tracking/delivery.md (hunk 1 above)
Comment (Astarsys) [reply docs/backend-api/resources/tracking/delivery.md:49]: the link to the descriptions below. I added a real example in a new commit

File: docs/backend-api/resources/tracking/geocoder.md (hunk 2 above)
Comment (Astarsys) [reply docs/backend-api/resources/tracking/geocoder.md:10]: done

File: docs/backend-api/resources/tracking/geocoder.md (hunk 3 above)
Comment (Astarsys) [reply docs/backend-api/resources/tracking/geocoder.md:12]: removed `Geocoder types:`

File: docs/backend-api/resources/tracking/retranslator.md (hunk 4 above)
Comment (Astarsys) [reply docs/backend-api/resources/tracking/retranslator.md:52]: done

**Comment by Astarsys** [comment]
All requested changes made

**Review by fenik17 — APPROVED** [review]

**Pull request merged**